### PR TITLE
Adapt loglevels to reduce log-noise

### PR DESCRIPTION
**Pull request by nupplaphil**
My logs are full with
```
opensocial-app-1  | Friendica[7347]: index [NOTICE]: created item {"post-id":51662648,"uid":0,"network":"apub","uri-id":35399433,"guid":"14550970-01f6d25ee2aea4bb-a324207f"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"717e287785d87ab8630e3c71bab9ab67","uid":"99
3191"}                                                                                                                                                                                                                                                                                      
opensocial-app-1  | Friendica[5159]: index [NOTICE]: created item {"post-id":51662649,"uid":711,"network":"apub","uri-id":35399432,"guid":"ae157d21-4f40a901ba7807d3-2b59a001"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"8c55503ab34e22df4f6193fb5c98163e","uid":"
a3042d"}                                                                                                                                                                                                                                                                                    
opensocial-app-1  | Friendica[7347]: index [NOTICE]: created item {"post-id":51662650,"uid":851,"network":"apub","uri-id":35399433,"guid":"14550970-01f6d25ee2aea4bb-a324207f"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"717e287785d87ab8630e3c71bab9ab67","uid":"
993191"}                                                                                                                                                                                                                                                                                    
opensocial-app-1  | Friendica[7347]: index [NOTICE]: created item {"post-id":51662651,"uid":1137,"network":"apub","uri-id":35399433,"guid":"14550970-01f6d25ee2aea4bb-a324207f"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"717e287785d87ab8630e3c71bab9ab67","uid":
"993191"}                                                                                                                                                                                                                                                                                   
opensocial-app-1  | Friendica[7347]: index [NOTICE]: created item {"post-id":51662652,"uid":3320,"network":"apub","uri-id":35399433,"guid":"14550970-01f6d25ee2aea4bb-a324207f"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"717e287785d87ab8630e3c71bab9ab67","uid":
"993191"}                                                                                                                                                                                                                                                                                   
opensocial-app-1  | Friendica[7348]: index [NOTICE]: created item {"post-id":51662653,"uid":0,"network":"apub","uri-id":35399434,"guid":"962c3e10-2064-1753-65a3-c66406895775"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"8c2a180c0c29dd418d4acec6441b452e","uid":"
a1ca5f"}                                                                                                                                                                                                                                                                                    
opensocial-app-1  | Friendica[7348]: index [NOTICE]: created item {"post-id":51662654,"uid":1137,"network":"apub","uri-id":35393620,"guid":"076f5bde-b6c9d9d91c058330-5ae3bbff"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"8c2a180c0c29dd418d4acec6441b452e","uid":
"a1ca5f"}                                                                                                                                                                                                                                                                                   
opensocial-app-1  | Friendica[7348]: index [NOTICE]: created item {"post-id":51662655,"uid":1137,"network":"apub","uri-id":35393678,"guid":"076f5bde-774ee148c2964844-23e6ca5b"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"8c2a180c0c29dd418d4acec6441b452e","uid":
"a1ca5f"}                                                                                                                                                                            
```
or
```
opensocial-app-1  | Friendica[5159]: index [NOTICE]: Message 373ebf56-5964-1709-d808-196775361433 already exists for user 0 [] - {"file":"Diaspora.php","line":943,"function":"messageExists","request-id":"aaab4a11b49a02d0ada9192827df9ca6","uid":"478651"}
opensocial-app-1  | Friendica[5159]: index [NOTICE]: Parent item found: parent: 373ebf56-5964-1709-d808-196775361433 - user: 0 [] - {"file":"Diaspora.php","line":1228,"function":"parentItem","request-id":"aaab4a11b49a02d0ada9192827df9ca6","uid":"478651"}
opensocial-app-1  | Friendica[2773]: index [NOTICE]: created item {"post-id":51662692,"uid":3320,"network":"dspr","uri-id":35399449,"guid":"b00842c0a8ae013b36aa101b0efced44"} - {"file":"Item.php","line":1273,"function":"insert","request-id":"5c0dd7138b4f8b64fabbf5995fc3f8fc","uid":"0
eb565"}
```

for me, it seems like this is "noise" for normal running nodes. So I set the loglevel to `info()`.
My rules of thumb:
- `info` in case the behavior is "normal" (like start of methods) - the description of `info` says: "Interesting events."
- `notice` in case the behavior is "normal", but is a special case (like something isn't found what should be set) - the description of `notice` says: "Normal but significant events". Like the parent-id isn't set, or the author is missing, ...

@annando  - What do you say? do we really need that much log-entries for example for  "expected" creation of items?

### CODE REVIEW

**Comment by annando** [comment]
For support reasons we do need some of these entries. We should have got at least one entry for a created item. From time to time we have got requests like "I haven't received that post" (or something similar). Having this on notice-Level helps. But of course mostly we should log something like "parent not found" with "notice" and "parent found" with "info", since this is not so important to know.

**Comment by nupplaphil** [comment]
But wouldn't a proper DB query on tables like post return exactly the same information? Do we really log all item creations just in case?

**Comment by annando** [comment]
We had stuff that got deleted after creation and such stuff.

File: src/Model/Item.php [hunk 1]
```diff
@@ -1270,7 +1270,7 @@ public static function insert(array $item, int $notify = 0, bool $post_local = t
 			Post\ThreadUser::insert($item['uri-id'], $item['uid'], $item);
 		}
 
-		Logger::notice('created item', ['post-id' => $post_user_id, 'uid' => $item['uid'], 'network' => $item['network'], 'uri-id' => $item['uri-id'], 'guid' => $item['guid']]);
+		Logger::info('created item', ['post-id' => $post_user_id, 'uid' => $item['uid'], 'network' => $item['network'], 'uri-id' => $item['uri-id'], 'guid' => $item['guid']]);
```
Comment (MrPetovan) [inline src/Model/Item.php]: Per feedback.
```suggestion
		Logger::notice('created item', ['post-id' => $post_user_id, 'uid' => $item['uid'], 'network' => $item['network'], 'uri-id' => $item['uri-id'], 'guid' => $item['guid']]);
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Pull request merged**